src/streamlink/plugins/youtube.py

```diff
@@ -205,11 +204,21 @@ def _get_stream_info(self, url):
         if not video_id:
             return
 
-        params = {
-            "video_id": video_id,
-            "el": "player_embedded"
-        }
-        res = http.get(API_VIDEO_INFO, params=params, headers=HLS_HEADERS)
+        for el in ("detailpage", "embedded"):
+            params = {
+                "video_id": video_id,
+                "el": el
+            }
+            res = http.get(API_VIDEO_INFO, params=params, headers=HLS_HEADERS)
+            info_parsed = parse_query(res.text)
```

Comment:
`parse_query` is now called twice, here and

`return parse_query(res.text, name="config", schema=_config_schema)`

i don't know if i can pass `info_parsed` with `name="config", schema=_config_schema` somehow